src/main/resources/application.properties

```diff
@@ -303,6 +303,11 @@ classification-service.password=
 # Classification Job Timeout in minutes
 classification-service.job.abort-after-minutes=45
 
+# Strategy to retrieve status of a classification. False by default for backward compatibility.
+classification-service.job.status.use-jms=false
+
+# Queue containing the status of a classification. Blank by default for backward compatibility.
+classification-service.job.status.location=
```

Comment:
If this approach is approved, I'll log an INFRA ticket for our environments to use JMS. By default, Snowstorm will not use JMS.